server/migrations/20221206163421-add-share-url-slug.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    try {
+      await queryInterface.addColumn("shares", "urlId", {
+        type: Sequelize.STRING,
+        allowNull: true,
+        unique: true,
```

Comment:
Modified from `unique(teamId, urlId)` to `unique(urlId)` because if we load `https://<team.url>/<urlId>`, the only available information to fetch required entities is `urlId`. And to be able to uniquely identify a `Share`(and eventually a `Document`), we need the unique constraint on `urlId`.

Reply:
We do have the team domain/subdomain from the url and a utility exists to convert that to a team – `getTeamFromContext` which should be enough. I _think_ this is needed because otherwise you can just type in "common" terms in the input and find shared content that way across all the teams…

Reply:
Ahh, didn't know that. Thanks :+1: